toolchain/sem_ir/formatter.h

```diff
@@ -47,8 +67,7 @@ class Formatter {
     llvm::SmallVector<size_t> dependencies = {};
   };
 
-  // A scope in which output should be buffered because we don't yet know
-  // whether to include it in the final formatted SemIR.
+  // Redirects formatted output to a new `OutputChunk` for tentative output.
```

Comment:
This seems like it documents the constructor, rather than the struct as a whole. How about something like this?

```suggestion
  // All formatted output within the scope of this object is redirected to a
  // new tentative `OutputChunk`.
```

Reply:
Done